proposal-templates/0000-swift-template.md

```diff
@@ -3,24 +3,22 @@
 * Proposal: [SE-NNNN](NNNN-filename.md)
 * Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
 * Review Manager: TBD
-* Status: **Awaiting implementation**
-
-*During the review process, add the following fields as needed:*
-
+* Status: **Awaiting implementation** or **Awaiting review**
+* Bug: *if applicable* [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN)
+* Feature Identifier: *if applicable; see SE-0362* `FeatureIdentifier`
```

Comment:
QTP: Is it useful here to flag a difference between experimental vs. upcoming features here?

My gut feeling is no, that the feature identifier alone suffices because whether a feature is experimental or upcoming is an artifact of the implementation—the author and the working group and/or release managers would determine whether a feature is stable enough to be considered upcoming instead of experimental. The proposal itself would care only about the identifier.

Reply:
Yes, I think some combination of proposing it and submitting the implementation to technical review is what makes something a non-experimental feature, and that's beyond the scope of Evolution.  The Language Workgroup should be monitoring it as part of its responsibility to report what work is available in each release, though.